### ISSUE TITLE
Allow microphone not working.

### ISSUE THREAD

**Issue by wolfsilver00** [amanharwara/altus#314]
On archlinux, altus 5.2

When trying to send a voice message, when clicking on the microphone button it asks for permission (as is expected) but when clicking on "Ok, got it", when there usually is a pop-up on the browser asking you to allow the microphone to run on that site, there is no pop-up, and the microphone button stops responding. There is no way (seemingly) to activate the microphone.

Altus: 5.2.0
Electron: 28.3.3
Chrome: 120.0.6099.291
V8: 12.0.267.19-electron.0
OS: Linux x64 6.9.3-arch1-1

**Comment by BattouSaeen** [comment]
Yup, btw @wolfsilver00 any idea which earlier version still supports it, if it all?

**Comment by wolfsilver00** [comment]
@BattouSaeen sorry man, I downgraded a lot and didn't find a way to make it work.. There was a comment somewhere with a particular version that said that it worked but I could not even log in in that one

**Comment by Sant1xCF** [comment]
Yeah so... the issue still persists... this is an importante feature...

**Comment by wolfsilver00** [comment]
> Yeah so... the issue still persists... this is an importante feature...

If you dont need the multiple tabs, ive gone to whatsie and it works without issue

**Comment by Sant1xCF** [comment]
tabs are the whole point. there is pretty much nothing else? will check out whatsie though

Em quinta-feira, 5 de setembro de 2024 às 20:20, wolfsilver00 ***@***.***> escreveu:

**Comment by NicolasPL64** [comment]
still no word from the devs about this bug. Really annoying tbh

**Comment by amanharwara** [comment]
https://github.com/amanharwara/altus/releases/tag/5.4.0 should fix this, please give it a try.

**Comment by BattouSaeen** [comment]
> https://github.com/amanharwara/altus/releases/tag/5.4.0 should fix this, please give it a try.

It works now, though I notice choppy recording. It's not an issue with my equipment because if I open any other recording application, I don't get the artefacts.

**Comment by wolfsilver00** [comment]
> > https://github.com/amanharwara/altus/releases/tag/5.4.0 should fix this, please give it a try.
> 
> It works now, though I notice choppy recording. It's not an issue with my equipment because if I open any other recording application, I don't get the artefacts.

@amanharwara can confirm, running the appimage I can get the microphone to work but this also happens to me
